css/global.css

```diff
@@ -724,9 +808,78 @@ Feel free to change the min/max-width values to whatever you need. Use variables
 }
 
 /* Tablet and smaller 768px */
-@media screen and (max-width: var(--mq-width-tablet-max)) {
+@media screen and (max-width: 768px) {
+	/* 
+	===================================
+	Modal Adjustment
+	===================================
+	*/
+	.modal-content {
+		width: 90%;
+		padding: var(--spacing-sm);
+	}
 }
 
+/* custom breakpoint 470px */
+@media screen and (max-width: 470px) {
```

Comment:
I have been trying to use --mq-width-phone-max, but it does not register during tests. 470px is working well as a  mobile breakpoint, let's consider 470px vs. 400px